code/game/machinery/cloning.dm

```diff
@@ -280,6 +280,7 @@
 		H.set_cloned_appearance()
 
 		H.set_suicide(FALSE)
+	to_chat(H, "<span class='warning'><b>You are being cloned. As per our rules, you cannot remember how you died.</b></span>") //NSV13
```

Comment:
```suggestion
	to_chat(H, "<span class='warning'><b>You are being cloned. You cannot remember how you died.</b></span>") //NSV13
```

Reply:
I thought adding the rules would make it very fucking clear that this is punishable, but if you think we should keep it IC then that's fine.